articles/5-real-life-use-cases-of-natural-language-processing-(nlp)/index.md

```diff
@@ -0,0 +1,59 @@
+Language as a method of communication helps people write, read, and speak. Individuals make plans and decisions in natural language, particularly in words. But the current artificial intelligence era is very determined to establish if individuals can communicate in a similar manner with computers. With the goal to ensure that human beings communicate with computers in their natural language, computer scientists have developed natural language processing (NLP) applications. For computers to understand the unstructured and often ambiguous human speech, they require the input of NLP applications.
+
+### Understanding Natural Language Processing
+
+The 21 st century is characterized by automation of systems and gadgets. Moreover, the use of computation is moving to elevated levels. The automation of things that we use daily is targeted to make our lives easier and better. Examples of products of automation include self-driving cars, google assistants, and artificial intelligence in systems. Each of these technologies has deep learning, machine learning or artificial intelligence technologies. Today people are embracing gadgets that rely on machine learning or artificial intelligence, due to their popularity in use in the field of data science, cloud computing, natural language processing, data engineering, computer vision, and more.
+
+Natural Language Processing (NLP) is a type of AI that seeks to enable computers to process or understand human language. Ideally, NLP does this by programming computers for analyzing and processing large quantities of natural language data. It manipulates speech, text through computational power enabled by the use of various software.
+
+Natural Language Processing (NLP) vs Natural Language Understanding (NLU) vs Natural Language Generation (NLG)
+
+Natural language processing contains algorithms hat help in speech recognition. NLP systems also rely on neural networks to classify texts, answer questions, perform sentimental analysis, among other tasks. A part of NLP is natural language understanding (NLU). In NLU, the program understands, finds meaning and performs a sentimental analysis. This is made possible with the program breaking the language down into small bits that are easily understandable. It also involves an analysis of the text, both syntactically and semantically. Once a smart device, search function or chatbot understands the language it has to talk back in a way that users will understand. This relates to natural language generation (NLG), and its work is to take data from a search result and turn it into understandable language.
```

Comment:
Sentimental Analysis is just one task under NLP and NLU. There are many other tasks under NLU as well. For example, context extraction, entity-relation extraction,etc. I feel this statement is factually incorrect and needs to be rewritten.